### ISSUE TITLE
RxJs for Angular Developers: Using HttpModule

### ISSUE THREAD

**Issue by mustafamg** [ReactiveX/rxjs-docs#155]
Many angular developers convert observer returned for HTTP request to promise before using them. This guidance will introduce to the developer how to declaratively use the returned observers instead of transforming it to promote.

**Comment by Only1MrAnderson** [comment]
The Angular Docs have a great example of this... https://angular.io/guide/http#making-a-request-for-json-data 
```
    this.http.get('/api/items').subscribe(data => {
      // Read the result field from the JSON response.
      this.results = data['results'];
    });
```

**Comment by mustafamg** [comment]
Yes, this one way of doing that. The other way is returning the observer to the template instead of the results coming from the subscription.
The other way, is as follow:
` this.results  = this.http.get('/api/items').map(data => data['results']);`
